### DISCUSSION TITLE
Seasonal MMR Reset

### DISCUSSION THREAD

**Discussion by cedricchi** [ValveSoftware/Dota2-Gameplay#19953] [Ideas]
As Team Spirit suggested it is time to radically overhaul the MMR system in Dota 2 and matchmaking after 11 years.
https://hawk.live/posts/team-spirit-suggested-that-valve-radically-overhaul-the-mmr-system-in-dota-2
1. Resetting MMR will revive Dota 2, as players will be climbing ranked again every season. Almost all other competitive games have seasonal reset except Dota 2.
2. Resetting MMR will dissuade all boosters and account buyers because their work/ money spent would be for nothing.
3. Match quality will improve when more players are in the pool, followed by better environment and less toxic community.
We must get rid of the very obvious forced 50% winrate and replace with random matchmaking because randomness = fairness. Account buyers are beyond comfortable with the forced 50%, that is why Dota 2 is warm bed for boosters and account buyers. Today Dota 2's matchmaking system has gone crazy, evidence include but not limited to 11 games losing streak then 11 games win streak. https://steamcommunity.com/profiles/76561198069167454/screenshot/2420194856766738645/
Yes, the number is 11. I have had two of such losing streaks as you can see from the screenshots. Exactly 11.
Only a rigged system and a failed one can do this to its money paying loyal players. THIS IS OUTRAGEOUS, nothing you can do when you lose. Trolls, feeders, jungle bots everything you can imagine are on my team making my games unplayable and stressful. I really need a strong mind to play Dota 2 which makes me wonder: why do I pay for something that forces me to eat poo poo?
@jeffhill @gaberowe @jeremyselan @lostgoat @bjj @bgilmore

**Comment by evil-inside1332** [comment]
i have an idea for a complete overhaul to the MMR system that i will post in a few weeks after i work out a few more kinks

boosting will not be possible, and griefing will be heavily penalized directly from the system

coming soon!
